index.js

```diff
@@ -14,7 +14,7 @@ function hoodieMiddleware(config) {
   var appConfig = this.project.config(config.options.environment);
   /* jshint +W040 */
 
-  if (!appConfig.hoodie) {
```

Comment:
it’s been a while since I looked at the code, but I think we should check for both?

```js
if (!appConfig.hoodie || !appConfig.hoodie.server) {
```

what do you think?